homeassistant/components/motion_blinds/__init__.py

```diff
@@ -57,6 +59,7 @@ def __init__(
         )
 
         self._gateway = coordinator_info[KEY_GATEWAY]
+        self._api_lock = coordinator_info[KEY_API_LOCK]
```

Comment:
```suggestion
        self.api_lock = coordinator_info[KEY_API_LOCK]
```

I think it would make sense to expose this attribute and then use it as `self.coordinator.api_lock` in the entity instead of passing it in

Reply:
much cleaner, thanks!